### PR TITLE
chore: describes logging with JARS and add a log shipper to docker-comp.

### PR DESCRIPTION
**Pull request by jekutzsche**
* There is a new container in the compose file with shipps all log
messages of the other running containers to log files in the configured
volume folder.
* The folder for the log files is configurable in .env.
* Fixes wrong log level setting: de.healthIMIS is outdated.

### CODE REVIEW

**Comment by hey-johnnypark** [comment]
As discussed with @jekutzsche , I will cherry pick his changes because I am currently refactoring the iris-client configuration.

File: infrastructure/deployment/docker-compose.yml [hunk 1]
```diff
@@ -157,6 +163,28 @@ services:
...
+        max-file: "5"
+    environment:
+      - LOG_FILES=true
+      - LOG_SYSLOG=false
+      - EXCLUDE=deployment_logger_1
+      - MAX_FILES=10
+      - MAX_SIZE=50
+      - MAX_AGE=28
+      - DEBUG=false
+#      - TIME_ZONE=Europe/Berlin
```
Comment (tboegner) [inline infrastructure/deployment/docker-compose.yml:183]: Why not use time zone "Berlin"?

**Review by tboegner — DISMISSED** [review]
Works fine, only suggestions in comments.

File: infrastructure/deployment/docker-compose.yml (hunk 1 above)
Comment (jekutzsche) [reply infrastructure/deployment/docker-compose.yml:183]: I had taken this from the template and have not found out where this affects.

**Pull request merged**